src/net.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include <net.h>
 
+#include <attributes.h>
```

Comment:
No longer needed since you removed `NODISCARD`

Reply:
Fixed!